x12-edi/2.9/modules/ROOT/pages/x12-edi-connector-reference.adoc

```diff
@@ -128,21 +272,25 @@ Transform an input stream of EDI text into the maps and lists structure of the E
 
 `<x12:write>`
 
-Transform the maps and lists structure of EDI data into an EDI text stream.
+Transforms the maps and lists structure of EDI data into an EDI text stream.
 
 
 === Parameters
 
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| Write Content a| Object |  |  `#[payload]` |
-| Streaming Strategy a| * repeatable-in-memory-stream
-* repeatable-file-store-stream
-* non-repeatable-stream |  Configure if repeatable streams should be used and their behavior. |  |
-| Target Variable a| String |  The name of a variable that stores the operation's output. |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and to store the outcome of that expression in the target variable. |  `#[payload]` |
+| Configuration | String | Name of the configuration to use. | |x
+| Write Content a| Object | Output payload |  `#[payload]` |
+| Streaming Strategy a| Enumeration, one of:
+
+* <<repeatable-in-memory-stream,Repeatable In Memory Stream>>
+* <<repeatable-file-store-stream,Repeatable File Store Stream>>
+* non-repeatable-stream
+| Configures how Mule processes streams. 
+|  | Repeatable file store stream |
```

Comment:
remove Repeatable file store stream box as this table is not rendering correctly